src/one_dragon_qt/view/code_interface.py

```
@@ -49,16 +48,16 @@ def __init__(self, ctx: OneDragonEnvContext, parent=None):
         v_layout = VBoxLayout(content_widget)
         v_layout.setAlignment(Qt.AlignmentFlag.AlignTop)
 
+        self.auto_update_opt = SwitchSettingCard(
+            icon=FluentIcon.SYNC, title='自动更新', content='使用exe启动时，自动检测并更新代码',
```

Ruff (0.14.3)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)